open-source.md

```diff
@@ -0,0 +1,67 @@
+
+ 1. **All code and assets necessary to operate the service are shared under an open source license, and publicly accessible.**
+ 1. **A public contributor can use the same workflow as a typical team member to make a change to the service.**
+
+For many projects the above seem obvious; this is how we generally work together on Open Source projects. Nevertheless such requirements are being explicitly specified, since services are a new area of work, and since the services and their sources are often not “distributed” for users to operate, and optionally modify, their own copy.
+
+## Frequently Asked Questions:
+
+Due to the variety of services, projects, teams and workflows there is a lot of flexibility in the requirements. Secondly, the requirements are a bare minimum, and best practices usually go beyond the requirements. Thus we must address common questions about the requirements:
+
+### Q: What “assets” should be Open Source?
+
+**A:** Make everything open by default, except where law, security, privacy or common sense says otherwise. A general rule of thumb is if you commit it to a repository, it should be open. Assets that should not be shared, are ones where you don’t have redistribution rights (such as a RHEL VM image, or private information, personal identifiable information), or things that are actually secret: Don’t share keys, passwords or tokens.
+
+If changing configuration is an activity that a typical team member does in order to change the service, then you should share that configuration publicly, in order to meet **requirement #2**.
+
+It is *recommended* that you include sufficient configuration to operate the service, or some base default version of configuration. Putting secrets in your configuration is a bad practice that should be avoided, regardless of Open Source.
+
+### Q: Should tests be shared with the code?
+
+**A:** Tests that are used during operation the service (as per **requirement #1**), and tests that used when run when making a change to the service (as per **requirement #2**), *must be shared publicly* under an Open Source license.
+
+It is *recommended* that tests which are independent of the service (eg: penetration tests or performance tests) are Open Source, but this is not required.
+
+### Q: Should anyone be able to trigger CI or test suites?
+
+**A:** The **requirement #2** above does *not require* that any public contributor can trigger the test suite, if that activity is limited to certain known contributors.
+
+It is common for Open Source projects to limit which contributors can trigger CI or run test suites on their provided infrastructure. This is usually done to prevent abuse such as exfiltrating secrets or denial of service. It’s *recommended* that your project has a mechanism whereby a known contributor, or maintainer, can review a contribution from an arbitrary contributor and then allow the CI or test suite to run on it.
```

Comment:
```suggestion
It is common for Open Source projects to limit which contributors can trigger CI or run test suites on their provided infrastructure. This is usually done to prevent abuse such as exfiltrating secrets or denial of service. It’s _recommended_ that your project has a mechanism whereby a known contributor, or maintainer, can review a contribution from an arbitrary contributor and then allow the CI or test suite to run on it.
```

Reply:
Updated to use underscore for italics.